libs/crossplane/upbound_aws_crds.libsonnet

```diff
@@ -29,8 +29,8 @@
   'apigateway.aws.upbound.io_integrationresponses.yaml',
   'apigateway.aws.upbound.io_integrations.yaml',
   'apigateway.aws.upbound.io_methodresponses.yaml',
-  'apigateway.aws.upbound.io_methodsettings.yaml',
   'apigateway.aws.upbound.io_methods.yaml',
+  'apigateway.aws.upbound.io_methodsettings.yaml',
```

Comment:
Interesting change, are you on a Mac? Seems like the `sort` command does something different. :)

Reply:
Yes I'm on Mac :).

Reply:
:shrug: It's fine.